### DISCUSSION TITLE
clone-wars

### DISCUSSION THREAD

**Discussion by giscus** [GorvGoyl/Personal-Site-Gourav.io#28] [Announcements]
Discussion related to the project:

**Title:** 100+ open-source clones and alternatives of popular sites like Airbnb, Instagram, Netflix, Tiktok, Spotify, Whatsapp, Youtube, etc. List contains source code, tutorials, demo links, tech stack, etc. Great for learning purpose!

**Link:** https://gourav.io/clone-wars

**Comment by ChrisEzike** [comment]
Thanks Gourav Goyal Kindly Keep the list updated we appreciate
